app/src/main/res/xml/preferences.xml

```diff
@@ -39,6 +39,14 @@
             android:key="absoluteTimeView"
             android:title="@string/pref_title_absolute_time" />
 
+        <ListPreference
+            android:defaultValue="default"
+            android:entries="@array/language_entries"
+            android:entryValues="@array/language_values"
+            android:key="language"
+            android:summary="%s"
+            android:title="@string/pref_title_language_settings" />
```

Comment:
Move the setting up. At least above the two SwitchPreferences, but I would even put it above the text size.
Also, please give it this icon: http://materialdesignicons.com/icon/translate